transaction/domain.go

```diff
@@ -0,0 +1,41 @@
+package transaction
+
+type TransactionRequest struct {
+	DebitAcc  string `json:"debit_acc"`
+	CreditAcc string `json:"credit_acc"`
+	Amount    int    `json:"amount"`
+}
+
+type TransactionWithDrawRequest struct {
+	DebitAcc string `json:"debit_acc"`
+	Amount   int    `json:"amount"`
+}
+type TrnasactionResponse struct {
+	Amount  int    `json:"amount"`
+	Message string `json:"message"`
+}
+
+type TransactionResponse struct {
```

Comment:
TransactionResponse -> Transaction


and also put rreemaining fields as well herre like ID etc